homeassistant/components/oralb/sensor.py

```diff
@@ -35,15 +35,19 @@
         device_class=SensorDeviceClass.DURATION,
         state_class=SensorStateClass.TOTAL_INCREASING,
         native_unit_of_measurement=UnitOfTime.SECONDS,
+        entity_category=EntityCategory.DIAGNOSTIC,
```

Comment:
Not sure about this one as it seems like the primary use case to tell how many seconds have been brushed (unless you are using it to turn on lights 😉)